src/__tests__/lib/containers/AccessApprovalsTable.test.tsx

```diff
@@ -101,7 +101,7 @@ describe('AccessApprovalsTable tests', () => {
     expect(
       screen.queryByRole('button', { name: 'Show More' }),
     ).toBeInTheDocument()
-    userEvent.click(moreButton!)
+    await userEvent.click(moreButton!)
```

Comment:
All userEvent APIs are now asynchronous. So most of the changes were adding `await`